docs/en/observability/profiling-use-a-proxy.asciidoc

```diff
@@ -0,0 +1,17 @@
+[[profiling-use-a-proxy]]
+= Use a proxy with Universal Profiling
+
+++++
+<titleabbrev>Use a proxy</titleabbrev>
+++++
+
+You can use the `HTTPS_PROXY` environment variable to configure a proxy used by the host-agent.
+The connection to the backend ({ecloud}) will be tunneled through the proxy (no MITM-TLS) using
+the CONNECT method. Basic authentication is supported. This can be useful in environments with security
+policies or network restrictions that block direct connections to outside hosts. 
+
+The following example shows how to use the `HTTPS_PROXY` environment variable:
+
+```
+export HTTPS_PROXY=http://username:password@proxy:port
+```
```

Comment:
Also here we should be more verbose that this setting need to be applied in the machines that will run the host-agents, and if the value is applied _after_ host-agent is installed, the host-agent installation may need to be restarted to take effect.

Reply:
Perfect thanks 👍🏼